src/functions/contactsMenu/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+type ContactsMenuCallback = () => void
+
+interface ContactsMenuHookProps {
+	id: string;
+	callback: ContactsMenuCallback;
+}
+
+declare global {
```

Comment:
Might end up in public types, so this global declarations should be in a `src/globals.d.ts` file and not something we export

Reply:
Ok, I'll move the global `Window._nc_[...]` definition and interface ` ContactsMenuHookProps` there.